asmcomp/cmxs_format.mli

```diff
@@ -0,0 +1,39 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*             Xavier Leroy, projet Gallium, INRIA Rocquencourt           *)
+(*                       Pierre Chambart, OCamlPro                        *)
+(*           Mark Shinwell and Leo White, Jane Street Europe              *)
```

Comment:
For what it's worth, the lines transferred to this file are Xavier's only, with minor edits from Gabriel

Reply:
Fixed